modules/dop/component-protocol/components/requirement-task-overview/chartBlock/burnoutChart/render.go

```diff
@@ -190,7 +192,14 @@ func (f *BurnoutChart) Render(ctx context.Context, c *cptype.Component, scenario
 			Series: []Series{
 				{
 					Data: func() []int {
-						counts := make([]int, 0)
+						counts := make([]int, 0, len(dates)+1)
+						counts = append(counts, func() int {
+							if h.GetBurnoutChartDimension() == "total" {
+								return sum
+							} else {
+								return sum / 60
+							}
+						}())
```

Comment:
separate a func, I noticed you also use this logic when build dateMap.

please add some unit test then.